pulpcore/cli/container/content.py

```diff
@@ -0,0 +1,93 @@
+import gettext
+from typing import Any
+
+import click
+
+from pulpcore.cli.common.context import PulpContext, PulpEntityContext, pass_pulp_context
+from pulpcore.cli.common.generic import (
+    GroupOption,
+    href_option,
+    list_command,
+    pulp_option,
+    show_command,
+)
+from pulpcore.cli.container.context import (
+    PulpContainerBlobContext,
+    PulpContainerManifestContext,
+    PulpContainerTagContext,
+)
+
+_ = gettext.gettext
+
+
+def _content_callback(ctx: click.Context, param: click.Parameter, value: Any) -> None:
+    if value is not None:
+        entity_ctx = ctx.find_object(PulpEntityContext)
+        assert entity_ctx is not None
+        if isinstance(entity_ctx, PulpContainerTagContext):
+            entity_ctx.entity = value
+        else:
+            entity_ctx.entity = {"digest": value}
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "content_type",
+    type=click.Choice(["blob", "manifest", "tag"], case_sensitive=False),
+    default="manifest",
+)
+@pass_pulp_context
+@click.pass_context
+def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> None:
+    if content_type == "manifest":
+        ctx.obj = PulpContainerManifestContext(pulp_ctx)
+    elif content_type == "tag":
+        ctx.obj = PulpContainerTagContext(pulp_ctx)
+    elif content_type == "blob":
+        ctx.obj = PulpContainerBlobContext(pulp_ctx)
+    else:
+        raise NotImplementedError()
+
+
+list_options = [
+    pulp_option("--digests", "digest_in", help=_("List of digests to filter {entities} results")),
+    pulp_option("--media-type"),
+    pulp_option("--names", "name_in", allowed_with_contexts=(PulpContainerTagContext,)),
```

Comment:
Doesn't this need to be 
```suggestion
    pulp_option("--names", "name__in", allowed_with_contexts=(PulpContainerTagContext,)),
```
in order to trigger the corresponding filter?